### PR TITLE
navbar and footer nav styled

### PR DESCRIPTION
**Pull request by bansara**
added sass (thanks dmytro!) to uiUpdate

styled navbar and footer
added roboto font
removed alert from nav
updated theme colors

### CODE REVIEW

File: src/components/navbar.tsx [hunk 1]
```diff
@@ -3,63 +3,73 @@ import { Link } from 'react-router-dom';
...
-            <li>
-              <Link to="/account" className={user.clearance ? 'ghost' : ''}>
-                Account
-              </Link>
-            </li>
-          </ul>
+              {!!user.clearance && (
+                <li className="user-profile">
+                  <Link to="/account">
+                    <img
```
Comment (DmytroLypovetskiy) [inline src/components/navbar.tsx:47]: can you use Avatar meterial UI component?
Please check https://github.com/SkrapLitter/capstone/pull/85/files#diff-d8e493dd70874bbfb8ad08ce303a16d0R91-R95

File: src/components/navbar.tsx [hunk 2]
```diff
@@ -3,63 +3,73 @@ import { Link } from 'react-router-dom';
...
+                  </Link>
+                  <div className="user-profile-subnav">
+                    <div>
+                      <button
+                        className="nav-btn"
+                        onClick={() => dispatch(logoutThunk())}
+                        type="button"
+                      >
+                        Logout
+                      </button>
```
Comment (DmytroLypovetskiy) [inline src/components/navbar.tsx:63]: Can we replace with UI component? https://github.com/SkrapLitter/capstone/pull/85/files#diff-2190f7797b70783925014d04daa36acdR155-R164

**Review by DmytroLypovetskiy — APPROVED** [review]

File: src/components/navbar.tsx (hunk 2 above)
Comment (bansara) [reply src/components/navbar.tsx:63]: the way this looks on the page is just like any other link in the navbar.
it's weird, i know, but if we try an onClick with an a tag, eslint is going to yell at us. that's why i did it this way.

Comment (DmytroLypovetskiy) [reply src/components/navbar.tsx:63]: hm. weird. didn't have this problems. anyway i have approved

**Pull request merged**